homeassistant/components/simplisafe/__init__.py

```diff
@@ -301,28 +261,29 @@ def v3_only(coro):
     @verify_system_exists
     @v3_only
     @_verify_domain_control
-    async def set_volume_property(call):
-        """Set a volume parameter in an appropriate service call."""
+    async def set_system_properties(call):
```

Comment:
Should this be an admin only service?

Reply:
I made it one because there are some properties that, if edited in a certain way, could constitute a security issue – for example, the service allows callers to turn the siren volume to `off`.

Reply:
Ok. I didn't remember that one of those decorators made it admin. My suggestion was to make it admin.

Reply:
`@_verify_domain_control` should take care of it.

Reply:
I don't see where it checks for admin:
https://github.com/home-assistant/home-assistant/blob/669844e4ddc7c3733ef03e4baf0e3ba6d1c5c3a6/homeassistant/helpers/service.py#L396-L439

I think we need this:
https://github.com/home-assistant/home-assistant/blob/669844e4ddc7c3733ef03e4baf0e3ba6d1c5c3a6/homeassistant/helpers/service.py#L373

Reply:
Ah, I got confused. Thanks!